data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1023 @@
+				accept
+	on complete
+		payment 400000
+	on visit
+		dialog `You left Teeneep's ship behind. Depart and wait for it to arrive before landing again.`
+	npc save accompany
+		government "Hai (Wormhole Access)"
+		personality timid escort
+		ship "Centipede (Teeneep)" "Teeneep's Ship"
+
+mission "Hai Rescue [B04] Diplomacy: Treaty Negotiation"
+	landing
+	name "Missing Researchers"
+	description "Escort Teeneep's Centipede to Parliament on <destination>. Payment is <payment>."
+	source "Earth"
+	destination "Sunracer"
+	on enter "Mirfak"
+		dialog `As you enter the system, authorities on <planet> ask you to land with Teeneep and meet a Syndicate representative immediately. They give you unusual landing coordinates, next to a warehouse.`
+	to offer
+		has "Hai Rescue [B03] Diplomacy: Syndicate Internal Affairs: done"
+	on visit
+		dialog `You left Teeneep's ship behind. Depart and wait for it to arrive before landing again.`
+	on offer
+		log `The President of the Republic and Chancellor of the Free Worlds met Teeneep, who claims to speak with the authority of the entire Hai government. They came to a tentative agreement about carefully tracking Hai movements in human space, but Teeneep won't agree until her people are returned. The human leaders asked us to talk to Stephen Moray again.`
+		conversation
+			`Earth authorities ask you to land in the Republic Capital Building, a vast skyscraper over a kilometer high which has its own spaceport. You land in one of several hangars built for ships significantly larger than a Carrier. In the distance, you spy an unfamiliar ship design which looks like a massive gunship. You've heard rumors that the Navy is working on advanced ships built to fight alien races. They may have left one here for Teeneep to see, as a warning to the Hai Elders.`
+			`	A formal delegation meets Teeneep at her ship, but she awaits you before continuing into the building. It takes about ten minutes of elevators and hallways to get to the top of the building: the Parliament Chamber. The chamber is empty; your delegation walks through it to a room on the far end of the chamber.`
+			choice
+				`	(Enter the room.)`
+					goto room
+				`	(Admire the scenery.)`
+			`	The entire outer wall of the 200-meter-wide room is a transparent metal which lets you see the steel forest of buildings that coat Earth. You're at the top of the tallest structure ever built by the human race.  The high altitude of the Parliament Chamber symbolizes its status in society: the supreme elected authority that oversees billions of people.`
+			`	Beyond the chamber resides the Crystal Room, perhaps the most famous room in the galaxy. It was there that monumental treaties were signed: the end of the Alpha Wars, the formal declaration of the illegality of slavery, the Syndicate and the Deep joining the Republic, the peace treaty with the Free Worlds, and numerous others.`
```

Comment:
This building almost certainly did not exist all that time ago. And, if it did, I have trouble believing that it is still the tallest humans have built.